acceptance/tests/resource/scheduled_task/should_create.rb

```diff
@@ -22,10 +22,10 @@
 
   step "verify task properties"
   on agent, puppet_resource('scheduled_task', name) do
-    assert_match(/command\s*=>\s*'c:\\windows\\system32\\notepad.exe'/, stdout)
+    assert_match(/command\s*=>\s*"c:\\\\windows\\\\system32\\\\notepad.exe"/, stdout)
     assert_match(/arguments\s*=>\s*'foo bar baz'/, stdout)
     assert_match(/enabled\s*=>\s*'true'/, stdout)
```

Comment:
The `enabled` property is stringified, because the `scheduled_task` type does not specify that it is boolean.